tests/neg-custom-args/isInstanceOf/3324h.scala

```diff
@@ -0,0 +1,10 @@
+object Test {
+  trait Marker
+  def foo[T](x: T) = x match {
+    case _: (T & Marker)       => // no warning
+    // case _: T with Marker => // scalac emits a warning
+    case _ =>
+  }
+
+  def bar(x: Any) = x.isInstanceOf[List[String]]  // error
```

Comment:
You could remove this line  and it `pos` test. This already tested in `i3324.scala`

Reply:
Thanks for the catch. This line is intended to make the test a neg test. Make it a pos test will not work, as we want to check warnings here, `-fatal-warning` is required. And it's nice to have all related tests of a feature under one folder.